extruct/__init__.py

```diff
@@ -1,16 +1,38 @@
+import logging
 from lxml.html import fromstring
-
 from extruct.jsonld import JsonLdExtractor
 from extruct.rdfa import RDFaExtractor
 from extruct.w3cmicrodata import MicrodataExtractor
+from extruct.opengraph import OpenGraphExtractor
+from extruct.microformat import MicroformatExtractor
 from extruct.xmldom import XmlDomHTMLParser
 
 
-def extract(htmlstring, url='http://www.example.com/', encoding="UTF-8"):
+logger = logging.getLogger(__name__)
+
+
+def extract(htmlstring, url='http://www.example.com/', encoding="UTF-8",
+            syntaxes="all", schema_context='http://schema.org'):
```

Comment:
what is schema_context argument for?

Reply:
it was needed by the tranformation part and forgot to take it away